Assignment03-Bootstrap/assets/scss/_colors.scss

```diff
@@ -0,0 +1,11 @@
+/*Defining color variables*/
+
+$dark-blue: #1977cc;
+$dark-navy-blue: #2c4964;
+$pure_white: #fff;
+$grey: #848484;
+$black: #343a40;
+$light-black: #ffffff33;;
+$light-blue: #f1f7fd;
+$light-grey: #d5e1ed;
+$blue: #8dc2f1;
```

Comment:
Please add a new line at the end of file

Reply:
Added a new line.